### DISCUSSION TITLE
malicious GitHub clone

### DISCUSSION THREAD

**Discussion by jeremymeadows** [community/community#17897] [Projects and Issues] [answered]
While searching for a repo via DuckDuckGo, I came across this other site that looks exactly like GitHub, other than a billion popup ads (surprisingly high up on the results page). I didn't look too much into what it's trying to accomplish, but it looks like the average "download our malware to fix your phone" kind of stuff. It was able to serve me several different repos by copy/pasting a valid path on the end of this url. I don't know if there is any sort of precedent for dealing with sites like this, but I figured that I would post it to raise awareness (be aware there were some 18+ ads present too): hxxp://rdok.ree.airlinemeals.net/content-https-github.com/

**Comment by ettaboyle** [comment, marked as answer]
Hi @jeremymeadows 👋🏻     

We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

* [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
* [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
* [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
* [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you,

The GitHub Community Forum Team
